spec/TestCustomObjectId.spec.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
Please move this test to an existing test file if tests for custom obj ID already exist